libc/test/src/math/smoke/LdExpTest.h

```diff
@@ -60,7 +71,7 @@ class LdExpTestTemplate : public LIBC_NAMESPACE::testing::FEnvSafeTest {
 
   void testOverflow(LdExpFunc func) {
     NormalFloat x(Sign::POS, FPBits::MAX_BIASED_EXPONENT - 10,
-                  NormalFloat::ONE + 0xF00BA);
+                  NormalFloat::ONE + 0xF00);
```

Comment:
`0xF00BA` is 20 bits, too large for `float16`'s 10-bit fraction.

(Comment moved from https://github.com/llvm/llvm-project/pull/94758.)